lib/views/login_signup.dart

```diff
@@ -0,0 +1,539 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+
+class login_signup extends StatefulWidget{
+  login_signup_state createState() => login_signup_state();
+}
+
+class login_signup_state extends State<login_signup>{
+
+  TextEditingController _txt1 = TextEditingController();
+  TextEditingController _txt2 = TextEditingController();
+  TextEditingController _username = TextEditingController();
+  TextEditingController _password = TextEditingController();
+  TextEditingController _email = TextEditingController();
+  TextEditingController _contact = TextEditingController();
+  FocusNode fnode1 = FocusNode();
+  FocusNode fnode2 = FocusNode();
+  FocusNode fnode3 = FocusNode();
+  FocusNode uname = FocusNode();
+  FocusNode password = FocusNode();
+  FocusNode email = FocusNode();
+  FocusNode contact = FocusNode();
+  final _formKey = GlobalKey<FormState>();
+  final _formKey2 = GlobalKey<FormState>();
+  late double h1,h2,h3; // this will be used for setting margin over waves
+  bool first_pane= true,onstart=true,display_text=true;// true if first page is open, false if close. onstart is only set once
+  bool visibility = true,visibility2 = true;
+
+  String? _validateEmail(String? value){
+    if(value!.isEmpty){
+      return "Please enter an email id";
+    }
+    String email_format = "[a-zA-Z0-9\+\.\_\%\-\+]{1,256}"+          //Regular Expression to match the email id
+        "\\@""[a-zA-Z0-9\\-]{0,64}"+
```

Comment:
Static strings should not be in the class. Create a separate category and declare all the static strings there.